src/components/new-request/search.vue

```diff
@@ -262,18 +262,23 @@ import { BulletsColinLink } from '../common'
 import NameInput from './name-input.vue'
 
 // Interfaces / Enums / List Data
-import { ConversionTypesI, EntityI } from '@/interfaces'
+import { ConversionTypesI, EntityI, FormType } from '@/interfaces'
 import { ActionBindingIF } from '@/interfaces/store-interfaces'
 import { AccountType, CompanyType, EntityType, Location, NrRequestActionCodes, NrRequestTypeCodes } from '@/enums'
 import { CommonMixin } from '@/mixins'
-import { CanJurisdictions, IntlJurisdictions } from '@/list-data'
+import { CanJurisdictions, ConversionTypes, Designations, IntlJurisdictions, RequestActions } from '@/list-data'
 import { GetFeatureFlag } from '@/plugins'
 
 /** This component is used for CREATING a Name Request. */
 @Component({
   components: { BulletsColinLink, NameInput }
 })
 export default class NewSearch extends Mixins(CommonMixin) {
+  // Refs
+  $refs!: {
+    selectBusinessTypeRef: FormType
```

Comment:
I added this to resolve a code warning about "form.reset() is not declared". Now that the code knows what this ref is, reset is declared.